### DISCUSSION TITLE
PDF Export version option

### DISCUSSION THREAD

**Discussion by dhawals1939** [jgraph/drawio-desktop#1911] [Q&A]
Hi, i use the tool to make figure for my overleaf.
Overleaf does not support PDF-1.7, If there is a chance that the figures can be exported with 1.4 or 1.5 that would be great.

**Comment by davidjgraph** [comment]
Looking at https://en.wikipedia.org/wiki/History_of_PDF PDF 1.7 came out in 2006 and is an ISO standard, whereas 1.5 isn't.

If Overleaf cannot deal with 1.7, it might well be an established tool, but we support an ISO standard version of PDF, Overleaf does not.
